tests/simulator_utils_test.py

```diff
@@ -7,32 +7,31 @@
 
 from sbi.inference.snpe import SnpeC
 from sbi.simulators.linear_gaussian import linear_gaussian
-from sbi.simulators.simutils import simulate_in_batches
+from sbi.inference.base import simulate_in_batches
```

Comment:
didnt `simulate_in_batches` end up back in `simutils`?

Reply:
Absolutely, thanks for catching this leftover.